src/components/Router/StepsRouter.js

```diff
@@ -20,19 +20,33 @@ class StepsRouter extends Component {
   currentComponent = () => this.props.componentsList[this.props.step]
 
   render = () => {
-    const { back, disableNavigation, isFullScreen, hideOnfidoLogo, options: { mobileFlow, ...globalUserOptions}, ...otherProps} = this.props
+    const {
+      back,
+      disableNavigation,
+      isFullScreen,
+      hideOnfidoLogo,
+      cobrand,
+      options: { mobileFlow, ...globalUserOptions },
+      ...otherProps
+    } = this.props
     const componentBlob = this.currentComponent()
     const CurrentComponent = componentBlob.component
     const options = componentBlob.step.options
     const stepId = `onfido-step${this.props.step}`  // to trigger update in NavigationBar on step change
-    // This prevents the logo from appearing late
-    const hideLogoLogic = mobileFlow ? hideOnfidoLogo : globalUserOptions.enterpriseFeatures?.hideOnfidoLogo && hideOnfidoLogo
+    // This prevents the logo, cobrand UI elements from appearing late
+    const hideLogoLogic = mobileFlow ?
+      hideOnfidoLogo :
+      globalUserOptions.enterpriseFeatures?.hideOnfidoLogo && hideOnfidoLogo
+    const cobrandLogic = mobileFlow ?
+      cobrand :
+      globalUserOptions.enterpriseFeatures?.cobrand
```

Comment:
Should this be `globalUserOptions.enterpriseFeatures?.cobrand && cobrand`? Or will that cause it to appear after the rest of the page has loaded? The problem with this is it allows users to use the cobrand feature even if they don't have the feature enabled on their account.

Reply:
I initially had it as `globalUserOptions.enterpriseFeatures?.cobrand && cobrand` but that had the side effect of causing it to appear only after the rest of the page has loaded so the default logo is still visible for a split second. Any suggestions for how to get around this?